data/categories.txt

```diff
@@ -4504,7 +4504,85 @@ sk:Nevestinec|verejný dom
 sw:Danguro
 fa:فاحشه خانه|جنده خانه
 
-amenity-casino|@entertainment|@nightlife
+@gambling
+en:4Gambling
```

Comment:
Remove 4, do not suggest it, search only for full name.

And I'd also remove all digits in casino, bookmaker, lottery, etc, if any.

Reply:
Done